src/app/static/src/app/components/modelOutput/ModelOutput.vue

```diff
@@ -262,7 +271,11 @@
                 }
                 // if unable to do the above, just updates the barchart as normal
                 this.updateBarchartSelections({payload})
-            }
+            },
+            prepareOutputDownloads: mapActionByName("downloadResults", "prepareOutputs")
+        },
+        mounted() {
+            this.prepareOutputDownloads();
```

Comment:
This and line 275 are the only changes, everything else in this file is a whitespace change as it had not been properly formatted. The change here is that preparing the output files is triggered on mount, i.e. when the user navigates to the "review outputs" step. This should be late enough that they have already calibrated the model to their satisfaction, but early enough that the files are ready for download by the time they navigate to the last step.

Reply:
Yeah, I think that's probably the case as they'd be expected to use the calibrate plot to check if calibration needed adjusting, thought I guess in some cases they may go back and re-calibrate, potentially before download files have finished preparing.

Reply:
Good point; ok, so as long as when resetting ids we also stop polling, should be fine.